### ISSUE TITLE
DECOHack throws an error when trying to set negative mass values for things

### ISSUE THREAD

**Issue by HALFKOOL** [MTrop/DoomTools#114]
Attempting to run the following code though DECOHack will result in the error below.
```
#include <doom19>
#include <friendly>

thing MTF_DEMON
{
	mass -1
}
```
ERROR: (C:\decohack\decohack.dh) Line 7, Token "}": Mass value (-1) not between 0 and 2147483647, inclusively.

**Comment by MTrop** [comment]
Hmm. I thought this was fixed in https://github.com/MTrop/DoomTools/issues/90. Will have to re-investigate.

**Comment by MTrop** [comment]
I was accepting a negative number, but not allowing it on range check. D'oh.

Fixed in commit ae3c74322cc279d59c367c5a84545fb3077d22a1.
